README.md

```diff
@@ -1,3 +1,39 @@
+# OMI Python Script Provider
+
+## How to build
+
+### Dependencies to build OMI Python Script Provider:
+- Install gcc and g++
+  - Redhat/CentOS:
+    ```
+    sudo yum update
+    sudo yum install gcc gcc-c++
+    ```
+  - Debian/Ubuntu OS:
+    ```
+    sudo apt-get update
+    sudo apt-get install gcc g++
+    ```
+- Install python and python-devel
+  - Redhat/CentOS:
+    ```
+    sudo yum update
+    sudo yum install python27 python27-devel
+    ```
+  - Debian/Ubuntu OS: 
+    ```
+    sudo apt-get update
+    sudo apt-get install python2.7 python2.7-dev 
+    ```
+    Note: If you use old OS version like Ubuntu 12.04 or Ubuntu 14.04, you might need to run command:`sudo add-apt-repository ppa:fkrull/deadsnakes` to get python to be installed.
+
+### To clone the repository to build OMI Python Script Provider, issue the following commands:
+```
+git clone --recursive https://github.com/Microsoft/Build-omi-script-provider
+cd Build-omi-script-provider/scriptprovider

```

Comment:
These instructions are missing a couple of steps here (https://microsoft.sharepoint.com/teams/OSTC/_layouts/15/WopiFrame2.aspx?sourcedoc=%7B8919B40F-858A-4B12-B6A8-3048436B61D7%7D&file=Script%20Provider%20Getting%20Started.docx&action=default):
    > cd build-omi-script-script-provider/omi/Unix
    > ./configure --dev
    > make
    > make install
    > cd ../../scriptprovider
    > make
    > sudo make install

Reply:
Got it, will update it, thanks.